engines/cryo/eden.cpp

```diff
@@ -4121,6 +4128,9 @@ void EdenGame::run() {
 			_musicPlayingFlag = false;
 			_musicEnabledFlag = false;
 		}
+
+		if (_vm->getPlatform() == Common::kPlatformMacintosh)
```

Comment:
Why is this limited to Mac platforms?

Reply:
The relevant structures are only allocated in `initCubeMac()`, which is only called for the Mac version.